lib/src/openApiToZod.ts

```diff
@@ -84,7 +84,12 @@ export function getZodSchema({ schema: $schema, ctx, meta: inheritedMeta, option
     if (schema.oneOf) {
         if (schema.oneOf.length === 1) {
             const type = getZodSchema({ schema: schema.oneOf[0]!, ctx, meta, options });
-            return code.assign(type.toString());
+            const chain = getZodChain({
+                schema: type.schema as SchemaObject,
+                meta: { ...meta, isRequired: true },
```

Comment:
I'm admittendly not entirely sure why `isRequired: true` is needed here and in the other places I added `getZodChain` calls. I tried excluding it at first, but it resulted in many test failures, and including it gave the results I was expecting.